lib/taurus/qt/qtgui/__init__.py

```diff
@@ -64,31 +63,36 @@ def __init__(self, name, package, entry_point):
 
     def __getattr__(self, member):
         import sys as __sys
-        mod = self.ep.load()
-        # Replace lazy module with actual module for package
-        setattr(__sys.modules[self.__package__], self.__name__, mod)
-        # Replace lazy module with actual module in sys.modules
-        modname = "%s.%s" % (self.__package__, self.__name__)
-        __sys.modules[modname] = mod
-        del __sys
-        return getattr(mod, member)
+
+        from taurus import warning as __warning
+
+        try:
+            mod = self.ep.load()
+            # Replace lazy module with actual module for package
+            setattr(__sys.modules[self.__package__], self.__name__, mod)
+            # Replace lazy module with actual module in sys.modules
+            modname = "%s.%s" % (self.__package__, self.__name__)
+            __sys.modules[modname] = mod
+            return getattr(mod, member)
+        except Exception as e:
+            __warning('Could not load plugin "%s". Reason: %s', __p.module_name, e)
+            return None
+        finally:
+            del __sys, __warning
```

Comment:
I think we can gain in clarity and resuability by moving the implementation of LazyModule into its own submodule (e.g. to `.../taurus/core/util/lazymodule.py`). This way it can be used for other entry points and we do not need to "pollute" the name space of this init.

Reply:
... and in that case there is no need of worrying about cleaning the namespace (and more natural import names could be used , e.g. `sys` instead of `__sys`, etc.)